### PR TITLE
Placer la déclaration des revenus du conjoint avant celles des enfants

### PR DESCRIPTION
**Pull request by fpagnoux**
Point remontés de test utilisateurs (via @HelaG ?)

### CODE REVIEW

File: app/js/controllers/foyer/ressources/ressources.js [hunk 1]
```diff
@@ -83,7 +83,12 @@ angular.module('ddsApp').controller('FoyerRessourcesCtrl', function($scope, $sta
         return result;
     };
 
-    $scope.individusVM = _.map($scope.situation.individus, function(individu) {
+    var demandeur = SituationService.getDemandeur($scope.situation);
+    var conjoint = SituationService.getConjoint($scope.situation);
+    var sortedIndividus = (conjoint ? [demandeur, conjoint] : [demandeur])
+        .concat(SituationService.getEnfants($scope.situation));
```
Comment (MattiSG) [inline app/js/controllers/foyer/ressources/ressources.js]:
```
[].concat(
    SituationService.getDemandeur($scope.situation),
    SituationService.getConjoint($scope.situation),
    SituationService.getEnfants($scope.situation)
).filter(function(individu) { return individu })
.map(function(individu) {
…
```

?  :)

Comment (fpagnoux) [inline app/js/controllers/foyer/ressources/ressources.js]: Done, thanks ! 
Not super confident about my indentation though

Comment (MattiSG) [inline app/js/controllers/foyer/ressources/ressources.js]: I would have simply put the last parenthesis on the same line as the dot (opening paren followed by newline, I expect a newline before the closing paren).
But this is very good as is  :)

**Comment by MattiSG** [comment]
GTM, good job!  :)

**Pull request merged**